expyfun/_externals/h5io/_h5io.py

```diff
@@ -0,0 +1,297 @@
+# -*- coding: utf-8 -*-
```

Comment:
you actually *only* need to include this file as `expyfun/_externals/_h5io.py`, I created the `h5io` module to be essentially a single file for this very purpose